### ISSUE TITLE
404 on npm install

### ISSUE THREAD

**Issue by Dreanmer** [graingert/slimerjs#33]
this can be the problem: `npm install slimerjs` tries to install `slimerjs-0.9.6-macosx.zip` but on download page the file is `slimerjs-0.9.6-mac.tar.bz2`

```
alexandre at alexandre in ~/projects/netbot
$ npm install slimerjs

> slimerjs@0.9.6-1 install /Users/alexandre/projects/netbot/node_modules/slimerjs
> node install.js

SlimerJS not found on PATH
Downloading https://download.slimerjs.org/releases/0.9.6/slimerjs-0.9.6-macosx.zip
Saving to /var/folders/bp/bt6dk1gs0bz_yyst49fvyvtw0000gn/T/slimerjs/slimerjs-0.9.6-macosx.zip
Receiving...

Error requesting archive.
Status: 404
Request options: {
  "uri": "https://download.slimerjs.org/releases/0.9.6/slimerjs-0.9.6-macosx.zip",
  "encoding": null,
  "followRedirect": true,
  "headers": {},
  "strictSSL": true
}
Response headers: {
  "date": "Thu, 03 Dec 2015 14:21:03 GMT",
  "server": "Apache/2.2.22 (Debian)",
  "strict-transport-security": "max-age=15768000",
  "vary": "Accept-Encoding",
  "content-length": "238",
  "connection": "close",
  "content-type": "text/html; charset=iso-8859-1"
}
Make sure your network and proxy settings are correct.

If you continue to have issues, please report this full log at https://github.com/graingert/slimerjs
npm WARN enoent ENOENT: no such file or directory, open '/Users/alexandre/projects/netbot/package.json'
npm WARN netbot No description
npm WARN netbot No repository field.
npm WARN netbot No README data
npm WARN netbot No license field.
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/5.1.0/bin/node" "/usr/local/bin/npm" "install" "slimerjs"
npm ERR! node v5.1.0
npm ERR! npm  v3.5.1
npm ERR! code ELIFECYCLE

npm ERR! slimerjs@0.9.6-1 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the slimerjs@0.9.6-1 install script 'node install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the slimerjs package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs slimerjs
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls slimerjs
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/alexandre/projects/netbot/npm-debug.log```

**Comment by graingert** [comment]
This was fixed in #30 try 0.9.6-2

**Comment by Dreanmer** [comment]
very thanks! @graingert
